charts/opensrp-web/templates/_helpers.tpl

```diff
@@ -82,15 +82,50 @@ REACT_APP_SENTRY_CONFIG_JSON: {{ (include "opensrp-web.sentryConfig" $scope ) |
 Get opensrp-webs sentry tags
 */}}
 {{- define "opensrp-web.sentryConfig" }}
-{{- $sentryMap := omit .Values.sentry "tags" -}}
+{{- $sentryMap := pick .Values.sentry "environment" -}}
 {{- $tagsMap := pick .Values.sentry "tags" -}}
 {{- $_ := set $tagsMap "release-name" (.Release.Name) -}}
 {{- $_ := set $tagsMap "release-namespace" (.Release.Namespace) -}}
+{{- if and .Values.sentry.sentryKey .Values.sentry.domain .Values.sentry.projectId -}}
+{{- $_ := set $sentryMap "dsn" (printf "https://%s@%s/%s" .Values.sentry.sentryKey .Values.sentry.domain .Values.sentry.projectId) -}}
+{{- end -}}
 {{- $sentryConfigs := merge $sentryMap $tagsMap -}}
 {{- $_ := set $sentryConfigs "release" .Values.image.tag -}}
 {{- $sentryConfigs | toJson -}}
 {{- end }}
 
+{{/*
+Define express headers config, helps with reformatting where needed.
+*/}}
+{{- define "opensrp-web.expressResponseHeaders" -}}
+{{- $scope := . -}}
+{{- $expressHeaderConfig := get .Values.express "expressHeaders" -}}
+{{- $reportTo := get $expressHeaderConfig "report-to" -}}
+{{- $otherHeaders := omit $expressHeaderConfig "report-to" -}}
+{{- if $reportTo -}}
+{{- $stringified := "" -}}
+{{- range $index, $reportEndpoint := $reportTo }}
```

Comment:
Do we expect multiple report to endpoints? If we only expect sentry then I feel like it would be easier to pass report-to as an object rather than array?

Reply:
yeah we could have several reporting endpoints, not just those that report csp violations.